calblueprint/meepanyar
# Netlify Deployment Redirects File

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
This PR introduces a `redirects` file so Netlify and react-router can work together.

Note: Because of CORs restrictions, if you want to log in and play with the app (not required for this PR review), you'll need to have the below chrome setting switched to disabled. This is because Chrome recently changed their cookie security default.

![image](https://user-images.githubusercontent.com/35501399/111682130-3e724680-87e1-11eb-98c3-3a26b74c3606.png)


[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review
1. Click the **Details** section of `netlify/meepanyar/deploy-preview — Deploy preview ready!` at the bottom of the PR
2. Should redirect you to `https://*-meepanyar.netlify.app/login`
3. Refresh at the `/login` endpoint and see that it brings you back as expected and doesn't present some error message

[//]: # 'The order in which to review files and what to expect when testing locally'

## Relevant Links
Got solution from: https://answers.netlify.com/t/netlify-page-not-found-when-sharing-react-router-dom-based-links/11744 

### Online sources

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
